crates/macro-support/src/parser.rs

```diff
@@ -365,9 +369,10 @@ impl<'a> ConvertToAst<(BindgenAttrs, &'a Option<String>)> for syn::ForeignItemFn
     fn convert(self, (opts, module): (BindgenAttrs, &'a Option<String>))
         -> Result<Self::Target, Diagnostic>
     {
-        let js_name = opts.js_name().unwrap_or(&self.ident).clone();
+        let default_name = self.ident.to_string();
```

Comment:
I used an explicit `let` here to make it live long enough. Is there a better way to do this? Now we're unconditionally allocating a string even though it might not be needed.